pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/FunctionRuntimeManager.java

```diff
@@ -130,6 +132,8 @@ public int size() {
     private final FunctionMetaDataManager functionMetaDataManager;
 
     private final ErrorNotifier errorNotifier;
+
```

Comment:
Why are we adding this here? I'm not seeing any usages

Reply:
Its not being used and will remove